typesystem/fields.py

```diff
@@ -658,6 +658,19 @@ def validate(self, value: typing.Any, *, strict: bool = False) -> typing.Any:
 
         return validated
 
+    def serialize(self, obj: typing.Any) -> typing.Any:
+        if obj is None:
+            return None
+
+        if isinstance(self.items, list):
+            return [
+                serializer.serialize(value)
+                for serializer, value in zip(self.items, obj)
+            ]
+
+        assert self.items is not None
```

Comment:
This assertion looks odd to me. Looks like `self.items` *can* legitimately be `None` here, no?

Reply:
You are right, thank you! I've fixed it.